examples/with-apivideo-upload/package.json

```
@@ -0,0 +1,28 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

```suggestion
    "start": "next start"
```

https://github.com/vercel/next.js/pull/36050/files#r872446770